### PR TITLE
Justin krs/customer detail

### PR DESCRIPTION
**Pull request by justinengle-krs**
Render customer details control on customer page

- Do some CSS organizing
- Add getUserById to display username
- Create a reusuable TextInput component
- Use class models on deserializing to allow for better control
- Create story

### CODE REVIEW

File: src/models/Customer/customer.ts [hunk 1]
```diff
@@ -1,8 +1,9 @@
-import type { Meta } from '@storybook/addon-svelte-csf';
+import type { Meta } from '../meta';
 import type { Source } from '../source';
 import type { User } from '../user';
+import { Address } from './address';
 
-export interface SystemCustomer {
+export class SystemCustomer {
```
Comment (matthampton-stock2shop) [inline src/models/Customer/customer.ts:6]: Is there a reason you decided to change these from interfaces to classes?

**Review by matthampton-stock2shop — COMMENTED** [review]
Going to merge back in to #66 and then on in to #65.

File: src/models/Customer/customer.ts (hunk 1 above)
Comment (justinengle-krs) [reply src/models/Customer/customer.ts:6]: Mostly a preference and standard we use.
I much prefer having more explicit control over how it deserializes from the JSON in the constructor and handling nulls. Do you have a preference?

Comment (matthampton-stock2shop) [reply src/models/Customer/customer.ts:6]: No - that's fine by me

**Pull request merged**